app/src/main/AndroidManifest.xml

```diff
@@ -78,6 +78,13 @@
                 <category android:name="android.intent.category.DEFAULT" />
                 <data android:mimeType="video/*" />
             </intent-filter>
+            <intent-filter>
+                <action android:name="android.intent.action.MAIN" />
+                <category android:name="android.intent.category.LAUNCHER" />
+            </intent-filter>
+            <meta-data
+                android:name="android.app.shortcuts"
+                android:resource="@xml/shortcuts" />
```

Comment:
you have to put the `meta-data` on the `SplashActivity` and remove the `intent-filter` otherwise the app will have two icons in the launcher.

Reply:
Moved the `meta-data` (also replaced the icon with the same as the launcher, for now?)